.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.REPL.enableREPLSmartSend": false
```

Comment:
@starsnatched is this needed for the `benchmark` command?

Reply:
No, it is not. That was from my visual studio code settings, you can disregard it. I forgot to remove that before pushing.